plone/scale/storage.py

```diff
@@ -14,6 +14,15 @@
 # See AnnotationStorage._keep_scale_seconds instead.
 KEEP_SCALE_MILLIS = 24 * 60 * 60 * 1000
 
+try:
+    long
+except NameError:
+    # Python 3 has no long, only int.
```

Comment:
any reason for not getting the python version and checking the first digit, as more or less is the usual way to check for that?